src/index.html

```diff
@@ -7,7 +7,48 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
```

Comment:
no needs to add a class selector since we can use html and body tags directly 
```suggestion
  <body>
```